CHANGELOG.md

```diff
@@ -1,29 +1,67 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
-See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
+See the [change log guidelines](https://github.com/halestudio/hale/compare/5.0.1...5.1.0) for information on how to structure the file.
+
+## [Unreleased](https://github.com/halestudio/hale/compare/5.0.1...5.1.0)
+
+## [5.1.0]
 
-## [Unreleased]
+### Added
+
+- Support for reading multiple sheets from an Excel file simultaneously
+- Enable exporting of multiple feature types to Excel
+- Allow loading Excel with blank rows and sheets for schema and source data
+- Add predefined and custom formatters for Date cells in Excel import
+- Enable exporting feature types with no data in Excel
+- Align Excel with CSV export wizard
+- Prefer properties with short paths as GeoJSON geometry property
+- Add wizard page to select the read mode for GeoJSON/JSON
+- Added support for importing data from JSON files, enhancing the versatility of data import options.
+- Introduced support for loading JSON schema, providing users with greater flexibility in handling JSON data structures.
```

Comment:
Introduced support for loading JSON schema --> Support for loading JSON schema

Reply:
We do not support loading [JSON Schema](https://json-schema.org/) as such, so generally I think this phrasing is misleading and may confuse users.

The term before ("Support loading schema from JSON data") was better. Could also be "Support extracting schema from JSON data"

Reply:
@stempler Thank you for providing the clarified description. I just wanted to draw attention to the use of "Introduced". Past tense is not used in any of the other descriptions- it would be good to align it throughout the document.